src/main/java/org/alfresco/utility/network/ServerHealth.java

```diff
@@ -64,16 +64,25 @@ public boolean isServerReachable(String server, int port)
         return reachable;
     }
 
-    public boolean isServerReachable() throws Exception
```

Comment:
I wanted this to throw exception so the test (BeforeSuite, BeforeClass) should fail fast